qiita_core/configuration_manager.py

```diff
@@ -117,6 +117,12 @@ class ConfigurationManager(object):
         The script used to start the plugins
     plugin_dir : str
         The path to the directory containing the plugin configuration files
+    help_email : str
+        The email address a user should write to when asking for help
+        Defaults to qiita.help@gmail.com
+    sysadmin_email : str
```

Comment:
Should these accept lists? I'm not sure but it might be helpful. BTW it's fine to leave like this.

Reply:
hm, the help_email is used to prompt the user via e.g. mailto: to one specific email address. I wouldn't know how an HTML element should deal with multiple addresses. And it would also confuse the user.

A list for the systadmin_email makes more sense to me. But I think the best way to inform multiple people about issues would be a mailing list or a shared mail account - which would boil down to one address here